test_collections/sdk_tests/support/python_testing/models/python_test_parser.py

```diff
@@ -163,6 +163,18 @@ def __parse_test_case(
     if pics_method:
         tc_pics = __retrieve_pics(pics_method)
 
+    # - PythonTestType.COMMISSIONING: test cases that have a commissioning first step
+    # - PythonTestType.NO_COMMISSIONING: test cases that follow the expected template
+    #   but don't have a commissioning first step
+    # - PythonTestType.LEGACY: test cases that don't follow the expected template
```

Comment:
can we move these comments (or copy it) to class PythonTestType(Enum) definition?

Reply:
Done